src/commands/env/transfer.js

```diff
@@ -0,0 +1,81 @@
+const isEmpty = require('lodash/isEmpty')
+
+const Command = require('../../utils/command')
+const { exit, log, logJson } = require('../../utils/command-helpers')
+
+class EnvTransferCommand extends Command {
+  async run() {
+    const { args, flags } = this.parse(EnvTransferCommand)
+    const { api, site } = this.netlify
+    const { siteA, siteB } = args
+
+    const siteId = {
+      from: site.id || siteA,
+      to: site.id ? siteA : siteB,
+    }
+
+    if (!siteId.to) {
+      log(
+        'Please include the site destination (siteB) as second argument, or try to run this command again inside a site folder',
+      )
+      exit(1)
+    }
+
+    const [siteFrom, siteTo] = await Promise.all([
+      api.getSite({ siteId: siteId.from }),
+      api.getSite({ siteId: siteId.to }),
+    ])
+
+    const [
+      {
+        build_settings: { env: envFrom = {} },
+      },
+      {
+        build_settings: { env: envTo = {} },
+      },
+    ] = [siteFrom, siteTo]
+
+    if (isEmpty(envFrom)) {
+      log(`${siteFrom.name} has no env variable, nothing to transfer`)
+      return false
+    }
+    // Apply environment variable updates
+    const siteResult = await api.updateSite({
+      siteId: siteId.to,
+      body: {
+        build_settings: {
+          // Merge from site A to site B
+          env: {
+            ...envTo,
+            ...envFrom,
+          },
+        },
+      },
+    })
+
+    if (flags.json) {
+      logJson(siteResult.build_settings.env)
+      return false
+    }
+
+    log(`Success transfer env variable from "${siteFrom.name}" => "${siteTo.name}"`)
+  }
+}
+
+EnvTransferCommand.description = `Transfer env from one site to another`
+
+EnvTransferCommand.args = [
+  {
+    name: 'siteA',
+    required: true,
+    description: '(From) site Id or Name, if second argument is ommited, this argument will be the destination (To)',
```

Comment:
Will this command work with a site name?

The API call accepts a site id.

Reply:
Ah i forgot about that, let me fix that

Reply:
I dont think it's possible, since the `getSite` method required a site Id to be called.

Reply:
I think we could just change the description to remove the name.
In a future PR we could support site names too by listing all sites and matching the name (out of scope for this PR).

Reply:
Yeah i think so, can i update the docs?

Reply:
`npm run docs` should do that for you once you update the description in the code.
If that doesn't work, just push the code changes and I'll sync the docs

Reply:
Okay, u can test them manually